### ISSUE TITLE
[Feature-Request] Notification when a certain user changes his/her online status 

### ISSUE THREAD

**Issue by zuernBernhard** [RocketChat/Rocket.Chat#8984]
### Description:

As a User I would like to be informed by notification if a certain user switches from offline to onlin e so that i can contact him/her directly.

**Comment by armand1m** [comment]
I think this is fairly simple to implement using the user presence module that already exists in the app. 

@zuernBernhard Do you have any detailed aspects of how you would like to see this working?

If we by default just send notifications to everyone, everytime someone changes their presence status, it could give an extremely poor UX.

I think this would be nice if you as a user could subscribe to some users status updates. We would have to build a section somewhere in the app to manage these subscriptions, it could be in the User profile, in Settings, even as a context menu when hovering on user avatar.

Any suggestions?

**Comment by zuernBernhard** [comment]
You are right. It would be great if it can be defined per user. eg right click -> configure alarms 

[x] Notify me if the user changes Status from [x] to [x] by play [x] sound
+ Add Notification

**Comment by armand1m** [comment]
It sounds like we would be spying on people but this seems also pretty neat 😅 

Ok, I'll try to take some time and get this feature implemented.

**Comment by s1rl3o** [comment]
Hi there,

our team would be glad to use this feature in assistfy.
Any news about implementation planning?

**Comment by marceloschmidt** [comment]
If someone wants to add this as a bot/app, we would add it to the marketplace.
